godotengine/godot
# Android: Add `POST_NOTIFICATIONS` permission to the list of permissions available in the Export dialog

Fixes https://github.com/godotengine/godot/issues/75673

As in subject

![Screenshot 2024-04-08 094705](https://github.com/godotengine/godot/assets/13484849/9cb8f98f-1265-49ab-9e0b-cadfc3bc4919)

The permission is in the `AndroidManifest` of the resulting app:

![Screenshot 2024-04-07 215701](https://github.com/godotengine/godot/assets/13484849/91eaebe2-099b-4c33-967a-5cf2055954e1)


## HOW TO TEST
- build the engine/Android export templates
- select the new notification permission with its checkbox
- export the apk
- inspect `AndroidManifest.xml` of the exported apk (you can drag/drop to Android Studio and check it there)
